service/controller/aws/v5/resource/awsconfig/current.go

```diff
@@ -2,11 +2,37 @@ package awsconfig
 
 import (
 	"context"
+
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Out of curiosity: Why did you choose `apismetav1`? I usually use `metav1` - I am wondering if there is a reason to use your version or if it's just preference.

Reply:
No, could be that this originates from examples for testing reactor usage or then we have this somewhere lurking. I'm fine with both forms. :)